client/src/components/Utils/Toast.tsx

```diff
@@ -0,0 +1,28 @@
+import { Toast as FlowbiteToast } from 'flowbite-react';
+import { HiCheck } from 'react-icons/hi';
+import { toast } from '../../redux/feats/globalSlice/globalSlice';
+import { useAppDispatch } from '../../redux/store';
+interface ToastProps {
```

Comment:
`nit:` Maybe change this to `Props` and to a `type` instead of interface to keep consistency with the other components created in the app?